README.md

```diff
@@ -57,7 +57,7 @@ chatbot.get_response("Hello, how are you today?")
 # Training data
 
 ChatterBot comes with a data utility module that can be used to train chat bots.
-At the moment there is three languages, English, Spanish and Portuguese training
+At the moment there are three languages— English, Spanish and Portuguese training
```

Comment:
I just noticed that this sentence it completely out of date. English, Spanish and Portuguese are no longer the only languages available in the training data. You're welcome to remove this sentence all together if you'd like to.

Reply:
@gunthercox Right! I've updated the PR. Please see if it suits. Thanks!